packages/httpsnippet-client-api/src/index.ts

```diff
@@ -88,7 +90,7 @@ const client: Client<APIOptions> = {
     link: 'https://npm.im/api',
     description: 'Automatic SDK generation from an OpenAPI definition.',
   },
-  convert: (source, options) => {
+  convert: ({ cookiesObj, headersObj, postData, queryObj, url, ...source }, options) => {
```

Comment:
This minor refactor is to match how client targets are built now upstream in HTTPSnippet: https://github.com/readmeio/httpsnippet/blob/main/src/targets/node/fetch/client.ts#L24